lib/src/Base/Solver/openturns/SolverImplementation.hxx

```diff
@@ -90,6 +90,10 @@ public:
                        const Scalar infValue,
                        const Scalar supValue) const;
 
+  /** Solve attempt to find one root to a system of equations function(x) = 0 given a starting point x_0 */
+  virtual Point solve(const Function & function,
+                      const Point & startingPoint) const;
```

Comment:
maybe we could add an extra OT::Interval argument (eventually empty by default) as the 1-d methods do

Reply:
I don't think that I fully understand what you mean here. Do you mean to add an infPoint and supPoint arguments in this method as well?

Reply:
an OT::Interval would be equivalent to adding infPoint/supPoint but would allow for infinite intervals too, also it can be passed directly to the OptimizationAlgorithm

Reply:
@jschueller done. For the moment two different methods but probably will use the hint of an empty interval